src/cluster_legacy.c

```diff
@@ -6532,3 +6866,55 @@ int clusterAllowFailoverCmd(client *c) {
 void clusterPromoteSelfToMaster(void) {
     replicationUnsetMaster();
 }
+
+/* Replicate all migrating and importing slot states to a
+ * a given replica client. */
+void clusterReplicateOpenSlots(void)
```

Comment:
Will drop my comment here, since this is roughly the point we were arguing over last time. The point of contention was if we should replicate this slot ownership information by the RDB stream (the replica will know what slots are being imported based on the replication stream) vs if it should be propagated exclusively through the clustering system (In that we should be exchanging importing metadata on the clusterbus). 

@soloestoy I suppose you were the main proponent of the clusterbus approach last time. The old core team decided to go with replication based approach, but since we have new folks we should re-evaluate our options.

Reply:
>I suppose you were the main proponent of the clusterbus approach last time. 

A bit more context. IIRC, @soloestoy's idea was to *replicate* the migrating/importing states via the cluster bus; while this PR *replicates* the two states as if they were the normal data.